gateway/webhook/webhook.go

```diff
@@ -193,10 +193,10 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 		if resp.statusCode != 0 {
 			code = resp.statusCode
 		}
-		pkgLogger.Debug(resp.err)
+		pkgLogger.Infof("IP: %s -- %s -- Response: %d, %s", misc.GetIPFromReq(r), r.URL.Path, code, resp.err)
```

Comment:
How often this branch can happen? Could it lead to excessive logs taking place?

Reply:
We should see them only on failures and helps in debugging.